topostats/validation.py

```diff
@@ -212,8 +212,10 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             "cmap": Or(
                 "afmhot",
                 "nanoscope",
-                error="Invalid value in config for 'plotting.cmap', valid values are 'afmhot' or 'nanoscope",
+                "gwyddion",
+                error="Invalid value in config for 'plotting.cmap', valid values are 'afmhot', 'nanoscope' or 'gwyddion'",
             ),
+            "mask_cmap": str,
```

Comment:
If adding a `dpi` option to `default_config.yaml` validation here would be required.

```suggestion
            "mask_cmap": str,
            "dpi": Or("figure", lambda n: 1 > 0, error="Invalid value for 'dpi', valid values are 'figure' or an int value > 0"),
```